_ng/full/generator.js

```diff
@@ -25,6 +25,7 @@ exports.MainGenerator = class MainGenerator {
       };
       let _repoHostUrl = {repoHostUrl: this.wrapper.repoHostUrl};
       let _server = this.wrapper.server;
+      let _test = this.wrapper.tests;
```

Comment:
Maybe since it's about the separation of the tests and not the existance of them, it'd be better if we called it `testsSeparated`? And then the boolean would make more sense in my opinion - being less confusing later on, I mean.